packages/main/test/specs/DatePicker.spec.js

```diff
@@ -809,13 +809,18 @@ describe("Date Picker Tests", () => {
 		await btnYear.click();
 		let displayedYear = await datepicker.getDisplayedYear(11);
 		assert.ok(await displayedYear.hasClass("ui5-yp-item--disabled"), "Years out of range are disabled");
+	});
+
+	it("Years are disabled when out of range, part 2", async () => {
+		const root = await datepicker.getRoot();
 		await root.keys("ArrowRight");
 
-		displayedYear = await datepicker.getDisplayedYear(10);
+		let displayedYear = await datepicker.getDisplayedYear(10);
 		assert.ok(await displayedYear.isFocusedDeep(), "Focus remained on year 2100");
 
 		displayedYear = await datepicker.getDisplayedYear(11);
 		assert.notOk(await displayedYear.isFocusedDeep(), "Years out of range (2101) can not be reached with keyboard");
+		await datepicker.closePicker();
```

Comment:
I noticed that in in DatePicker.spec.js we now add 
`await datepicker.closePicker()`

why this is needed now, and was not before

Reply:
The tests before would take advantage of the fact that `openPicker()` doesn't just open the picker, it resets the view back to day view. Now the view is reset on popover open. 

The method openPicker() in the test file only updates the date picker's open property, and since the picker is already open, the next call to openPicker() doesn't have any effect (which is to be expected).